pointcloud.js

```diff
@@ -474,13 +474,18 @@ count_loop:
     }
 
     var glyphData
+    var input = ''
     if(Array.isArray(glyphs)) {
-      glyphData = getGlyph(glyphs[i], font)
+      input  = glyphs[i]
     } else if(glyphs) {
-      glyphData = getGlyph(glyphs, font)
-    } else {
-      glyphData = getGlyph('●', font)
+      input  = glyphs
+    }
+    if((input === '') ||
+       (input.length === input.replace(/[^ ]/g, '').length)) { // check for all blank case
```

Comment:
When does `input` have multiple blank characters?

------

Moreovee, wouldn't [`\s`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#special-white-space) be safer here?

Reply:
In cases where there was no surface representation for the text we get this warning. That includes cases like this: `''` or `'\s'` or `'\s\s'` or `'\s\s\s'` etc.

Reply:
There are lots of whitespace characters. FWIW I did a deep dive into this a few years ago and `allBlankCharCodes`, ugly as it is, was far and away the fastest solution:

https://github.com/plotly/fast-isnumeric/blob/efdd602521132349539fcf32161f4f62f10ea741/index.js#L29-L42

Reply:
Thanks @alexcjohnson for the link. This is a great function. 
I think we should use it there. Right?

Reply:
Yes, might be worth making a new little module for it, ie split it out from `fast-isnumeric` as `fast-isblank` or something? Then we can reuse it anywhere...